mismi-s3/src/Mismi/S3/Data.hs

```diff
@@ -10,6 +10,8 @@ module Mismi.S3.Data (
   , Address (..)
   , Key (..)
   , ReadGrant (..)
+  , Bytes (..)
+  , Sized (..)
```

Comment:
Just so your aware the rest of these re-exports are purely to keep compatibility